app/views/tasks/new.html.erb

```diff
@@ -0,0 +1,23 @@
+<%= simple_form_for([@user, @task], wrapper: :vertical_range) do |f| %>
+  <%= f.error_notification %>
+  <%= f.error_notification message: f.object.errors[:base].to_sentence if f.object.errors[:base].present? %>
+  <h1 class="text-center"><%= t('task.action.add') %></h1>
+  <br>
+  <div class="form-inputs d-flex flex-column">
+    <%= f.input :title %>
+    <%= f.input :description %>
+    <%= f.input :content, as: :text %>
+    <%= f.input :user_ids, collection: User.all.map { |u| [u.username, u.id] }, input_html: { multiple: true } %>
```

Comment:
maybe assignees sound better than user_ids?

Reply:
Dont i have to create a new field on the tasks model then? Like this I can just save it without havig a new fiels because of the plural name. Correct my please if I'm wrong

Reply:
Simple form might have way of doing this.
Can you check this out? 🤔  https://stackoverflow.com/questions/31226462/rails-4-change-name-attribute-of-simple-form-input

Reply:
Can you try this ? 🤔 
```suggestion
    <%= f.input :user_ids, label: 'Assignees:', collection: User.all.map { |u| [u.username, u.id] }, input_html: { multiple: true } %>
```

Reply:
I did it... I think 😄 I reconfigured the Models and the Table. Actually it worked easier as I thought. Thanks for asking me about this whole issue. I did some research and do now fully understand the difference between has_many_and_belongs_to_many and has_many :through. Let me know if you have some further suggestions please.